pandas/tests/groupby/test_groupby.py

```diff
@@ -1714,3 +1714,36 @@ def test_groupby_multiindex_nat():
     result = ser.groupby(level=1).mean()
     expected = pd.Series([3., 2.5], index=["a", "b"])
     assert_series_equal(result, expected)
+
```

Comment:
test goes in test_function.py

Reply:
moved in https://github.com/pandas-dev/pandas/pull/25531/commits/d874ef7adf0b5fdad7525ed8b26164447ba9044f